src/models.rs

```diff
@@ -9,6 +12,22 @@ use serde::{Deserialize, Serialize};
 
 pub type ResultWithDefaultError<T> = Result<T, Box<dyn std::error::Error>>;
 
+pub struct Entities {
+    pub time_entries: HashMap<i64, TimeEntry>,
```

Comment:
This loses the order from backend, we either need to sort whenever we convert to a list or store the list itself instead.